.github/workflows/ci.yml

```diff
@@ -8,81 +8,74 @@ jobs:
     runs-on: ubuntu-latest
     # Map a step output to a job output
     outputs:
-      should-skip-job: ${{ steps.skip-check.outputs.should_skip }}
+      should-skip-job: ${{steps.skip-check.outputs.should_skip}}
```

Comment:
Fixed the spacing throughout this file, some had extra spaces, and others did not.